server/api/controllers/UserControllers.js

```diff
@@ -0,0 +1,58 @@
+const User = require("../models/User");
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+
+const secret = process.env.SECRET;
+
+function generateToken(user) {
```

Comment:
JWTs are generated on successful register or login and returned to the user in the HTTP response.